src/Files.App/ViewModels/ItemViewModel.cs

```diff
@@ -1101,7 +1101,10 @@ public async Task LoadExtendedItemProperties(ListedItem item, uint thumbnailSize
 										item.FileFRN = fileFRN;
 										item.FileTags = fileTag;
 									}, Microsoft.UI.Dispatching.DispatcherQueuePriority.Low);
-									FileTagsHelper.DbInstance.SetTags(item.ItemPath, item.FileFRN, item.FileTags);
+									using (var dbInstance = FileTagsHelper.GetDbInstance())
+									{
+										dbInstance.SetTags(item.ItemPath, item.FileFRN, item.FileTags);
+									}
```

Comment:
This bit of code is used three times at least. Worth to be wrapped into a method `private void SetTagsOnDBInstance(ListedItem item)` ?